src/github.rs

```diff
@@ -1093,7 +1094,11 @@ impl<'q> IssuesQuery for Query<'q> {
             .await
             .with_context(|| "Unable to get issues.")?;
 
-        let fcp_map = crate::rfcbot::get_all_fcps().await?;
+        let fcp_map = if include_fcp_details {
+            crate::rfcbot::get_all_fcps().await?
+        } else {
+            HashMap::new()
+        };
```

Comment:
Why don't you place `let fcp_map = crate::rfcbot::get_all_fcps().await?;` on line https://github.com/rust-lang/triagebot/pull/1624/files#diff-ef5fd0828e2727ce9e100f72dc4f18f0f126e393d35108269a9bea5e8a104d5cR1117